src/shader_recompiler/ir/opcodes.inc

```diff
@@ -142,6 +142,7 @@ OPCODE(FPRecip64,                                           F64,            F64,
 OPCODE(FPRecipSqrt32,                                       F32,            F32,                                                                            )
 OPCODE(FPRecipSqrt64,                                       F64,            F64,                                                                            )
 OPCODE(FPSqrt,                                              F32,            F32,                                                                            )
+OPCODE(FPInvSqrt,                                           F32,            F32,                                                                            )
```

Comment:
This opcode already exists (FPRecipSqrt32).

Reply:
Oops, opcode blind